docs/tutorial.md

```diff
@@ -3,200 +3,457 @@
 
-### Creating a Bot
+Javascript, often abbreviated to JS, is a programming language designed for the web. It is what makes most interactivity on the web possible.  
+Node.js, often just Node, makes it possible to use Javascript outside of web browsers.
 
-To get a feeling for the Mineflayer and how to use it, please open a text editor and follow the next few steps.
+So the first thing you have to do to get started is to install Node. You can get it [here](https://nodejs.org/en/download/).  
+After it is installed, open a command prompt (also known as a terminal) and then type `node -v`  
+If you have installed Node correctly, it should return a version number. If it says it can't find the command, try installing it again.
 
-First, create an instance of the Bot.
+Now you have Node, you could start writing code, but we need to do 1 more thing.  
+Javascript can be written in any basic text editor, but it is much easier if you use what is called an [Integrated development environment](https://en.wikipedia.org/wiki/Integrated_development_environment)(IDE)  
+An IDE will help you write code because it can give you suggestions, or tell you if your code has potential problems. A good IDE to start with is [Visual Studio Code](https://code.visualstudio.com/)(VSCode)  
+Once you have installed and set-up VSCode, create a new file and then save it somewhere with a name ending with `.js`, e.g. `bot.js`  
+This will let VSCode know we are working with Javascript, and give you the correct suggestions.
+
+#### Javascript variables
+
+Start by typing the following:
 
 ```js
-const mineflayer = require('mineflayer')
-const options = {
-  host: 'localhost', // optional / minecraft server address
-  port: 25565, // optional
-  username: 'email@example.com', // email and password are required only for
-  password: '12345678', // online-mode=true servers
-  version: false // false corresponds to auto version detection (that's the default), put for example '1.8.8' if you need a specific version
+var test = 5
+```
+
+This will create a new variable named `test` and assign it the value `5`  
+Variable are used to save data and use it later in the code.
+
+Now save the file so we can run the code. Open a terminal again (or a new terminal in VSCode) and navigate to the same folder the file is saved in. This can be done using the `cd` command, for example: `cd Documents\javascript`  
+Once your terminal is in the same folder as your Javascript file, you can run `node filename.js`  
+If you have done everything correctly, you should see nothing.
+
+#### Javascript functions
+
+Next you will learn about functions. Functions are a piece of code that can be used multiple times throughout your code.  
+These can be useful because you don't have to type something multiple times.
+
+```js
+function addition(a, b) {
+  var result = a + b
+  return result
 }
-const bot = mineflayer.createBot(options)
+
+var test1 = addition(5, 10)
+var test2 = addition(1, 0)
```

Comment:
Yeah that makes sense, done.

Reply:
You could have kept the other version, to show the 2 syntaxes next to each others and explain that it means the same thing.

Reply:
Wouldn't that just cause confusion? Isn't it better to only show the best syntax available?

Reply:
I'm ok with that. It's just that some of our code still uses the old syntax (and there is some special case where we cannot do otherwise) so users may encounter both syntaxes.

Reply:
I've added a line about the `function` syntax, without going to deep into it.

Reply:
Looks good :)